README.md

```diff
@@ -2,9 +2,9 @@
 Make Python great again
 
 ## Mission
-TrumpScript is a language based upon the illustrious Donald Trump. As the undeniably best presidential candidate in the 2016 language, we found that the current field of programming languages does not include any that Trump's glorious golden combover would approve of.
+TrumpScript is a language based upon the illustrious Donald Trump. As the undeniably best president ever, we found that the current field of programming languages does not include any that Trump's glorious golden combover would approve of.
```

Comment:
Nope. Unless he would get assassinated, he is president.
You can wait untill he actually becomes president though.